mix.lock

```diff
@@ -2,4 +2,5 @@
   "credo": {:hex, :credo, "0.8.6", "335f723772d35da499b5ebfdaf6b426bfb73590b6fcbc8908d476b75f8cbca3f", [:mix], [{:bunt, "~> 0.2.0", [hex: :bunt, repo: "hexpm", optional: false]}], "hexpm"},
   "earmark": {:hex, :earmark, "1.2.3", "206eb2e2ac1a794aa5256f3982de7a76bf4579ff91cb28d0e17ea2c9491e46a4", [:mix], [], "hexpm"},
   "ex_doc": {:hex, :ex_doc, "0.16.3", "cd2a4cfe5d26e37502d3ec776702c72efa1adfa24ed9ce723bb565f4c30bd31a", [:mix], [{:earmark, "~> 1.1", [hex: :earmark, repo: "hexpm", optional: false]}], "hexpm"},
-  "gherkin": {:hex, :gherkin, "1.4.0", "b7b6a80029725a4b29a1c58f96ddb3fa9409d4fa907503384b1f86392491fcd0", [:mix], [], "hexpm"}}
+  "gherkin": {:hex, :gherkin, "1.4.0", "b7b6a80029725a4b29a1c58f96ddb3fa9409d4fa907503384b1f86392491fcd0", [:mix], [], "hexpm"},
```

Comment:
What do you think of marking poison as `:optional`? It's only required for anyone actually using the json formatter

Reply:
Agreed, changed it.